autoload/db/adapter/redis.vim

```diff
@@ -7,7 +7,23 @@ function! db#adapter#redis#canonicalize(url) abort
 endfunction
 
 function! db#adapter#redis#interactive(url) abort
-  return ['redis-cli'] + db#url#as_argv(a:url, '-h ', '-p ', '', '', '-a ', '-n ')
+  let url = db#url#parse(a:url)
+  let cmd = ['redis-cli']
+  for [k, v] in items(url.params)
+    " Specifying only connection releated flag here, missing flags can be added later
+    if k =~# '^\%(cert\|key\|cacert\|capath\|tls-ciphers\|tls-ciphersuites\)$' && v isnot# 1
+      call add(cmd, '--' . k . '=' . v)
+    elseif v =~# '^[1Tt]$'
```

Comment:
This would make sense to have a `k` check too, no? That is `k ==# 'tls' && v =~# '^[1Tt]'`, or something similar.

Reply:
Is there an upside to doing it that way? The downside would be it could potentially pass a non-Boolean option as Boolean, resulting in a potentially confusing error message.

Reply:
No upside, just some laziness in handling it 😉 , will add the parameters mentioned in help doc explicitly.